firehose/info/info_filler.go

```diff
@@ -13,37 +13,37 @@ var DefaultInfoResponseFiller = func(firstStreamableBlock *pbbstream.Block, resp
 	resp.FirstStreamableBlockId = firstStreamableBlock.Id
 
 	shortTypeURL := strings.TrimPrefix(firstStreamableBlock.Payload.TypeUrl, "type.googleapis.com/")
-	for _, protocol := range wellknown.WellKnownProtocols {
-		if protocol.BlockType == shortTypeURL {
-			resp.BlockIdEncoding = protocol.BytesEncoding
+	for _, protocol := range wellknown.GetRegistryNetworks() {
+		if protocol.Firehose.BlockType == shortTypeURL {
+			resp.BlockIdEncoding = wellknown.BlockIdEncodingForNetwork(protocol)
 			break
 		}
 	}
 
 	if !validate {
 		if resp.ChainName == "" {
 			// still try to fill the chain name if it is not given
-			if chain := wellknown.WellKnownProtocols.ChainByGenesisBlock(firstStreamableBlock.Number, firstStreamableBlock.Id); chain != nil {
-				resp.ChainName = chain.Name
+			if chain := wellknown.ChainByGenesisBlock(firstStreamableBlock.Number, firstStreamableBlock.Id); chain != nil {
+				resp.ChainName = chain.ShortName
```

Comment:
@maoueh this is not exactly like it was in the code before (like `Ethereum` vs. `ethereum`). 
Does this have an impact? Because we check shortname, fullname and aliases in ChainByName, I'm guessing it's ok?